### ISSUE TITLE
Merge into lmdb

### ISSUE THREAD

**Issue by kallisti5** [bendiken/lmdbxx#28]
**Describe the bug**
This is one single header wrapped in an entire package.  Maybe it should just be merged into lmdb? They accept pull requests.

**Comment by hoytech** [comment]
Into the LMDB distribution proper, from symas? That is highly unlikely to ever happen, since it's a pure C project and I don't think they're very enthusiastic about C++.

Also, there are multiple C++ wrappers, so who is to say which one becomes the bundled version? I would impudently suggest that my fork of this project (https://github.com/hoytech/lmdbxx) be considered too. :)

**Comment by kallisti5** [comment]
Yours is definitely more updated.  I'm a package maintainer for Haiku, so trying to figure out which to use for our re-distribution.

**Comment by hoytech** [comment]
In my obviously biased opinion, if you are OK with requiring a C++17 or higher compiler, you should use my fork since it has some important bugfixes.

**Comment by kallisti5** [comment]
@hoytech ow.  Yours definitely isn't API compatible with this one.  Maybe you should have bumped the major version number vs minor :-)

**Comment by hoytech** [comment]
Good point, sorry about that. Some of the API changes are necessary. I believe removing the templated accessor methods was especially important, they frequently caused bugs.

Anyway I tagged a 1.0.0 release too:

https://github.com/hoytech/lmdbxx/commit/029fb681213af3020f5e3a86045445552bd024c5
